src/utils/thread_helper.hpp

```diff
@@ -302,6 +302,10 @@ class ThreadHelper
 
     void ActiveDatasetChangedCallback(void);
 
+#if OTBR_ENABLE_TELEMETRY_DATA_API && OTBR_ENABLE_BORDER_ROUTING
+    void GetExternalRouteInfo(threadnetwork::TelemetryData_ExternalRoutes *aExternalRouteInfo);
```

Comment:
nit: After a second look I think the `Get` method usually has a return value but this one seems more like filling in the field. Maybe call it `Fill` or `Retrieve`.

Reply:
+1 Populate/Fill/Retrieve seems to be a better name.

Reply:
Thanks for the suggestion @superwhd and @morningboata , I just renamed to `RetrieveExternalRouteInfo` in the latest push